utils/fhir2_atom_feed_config.json

```diff
@@ -187,7 +187,7 @@
       "openMrsClass": "org.openmrs.Obs",
       "enabled": "true",
       "title": "Observation",
-      "category": "observation",
+      "category": "obs",
```

Comment:
@pmanko do you know if this change will have any implications on atom-feed2fhir module

Reply:
I can't remember where/how `category` was being used but can we make sure that the Atom Feed based approach works with this change (e.g., adding new observations are properly picked by the Atom Feed based streaming pipeline)?

Reply:
Actually IIUC, changing this will change the URL for the Observation feed and hence our client side code should also be updated accordingly [here](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/035a2e806c8c60816aef4d512425ab4b070819f8/streaming/src/main/java/org/openmrs/analytics/FeedConsumer.java#L51). Can you check if with that change it works?
If my understanding is correct then I think the bigger issue is that relying on the same `category` field for both database table name and fhir resource (lower case) is not right. WDYT?

Reply:
I see. I think we should create a  separate JSON config for atom-feed so that we can avoid this issue.

Reply:
Yeah, either that or adding a new field (e.g., `table`) to the each config. Given #36 I feel that creating a new json config is probably the better approach because fields like `openMrsClass` are irrelevant here and their presence might be confusing.

Reply:
sounds good!